### DISCUSSION TITLE
How to configure for Azure DevOps PaaS

### DISCUSSION THREAD

**Discussion by symplrdudley** [BoxThatBeat/obsidian-agile-task-notes#23] [Q&A] [answered]
I think I followed the pretty intuitive parts correctly, but I am still having issues.

![Obsidian Azure-Jira Setup](https://github.com/BoxThatBeat/obsidian-agile-task-notes/assets/130775229/9346cdd5-a9aa-4432-a5d9-a35274fe5094)

**Section 1**
This section should be either `Azure DevOps` or `Jira`. I am attempting to hook into Azure DevOps, so I left it as default.

**Section 2**
The _Instance_ should be `https://dev.azure.com` I believe, but it could potentially be: `https://dev.azure.com/{Project Collection}`. Either way, this didn't seem to jive because I kept getting unknown name errors.

**Section 3**
Collection is what I figured the _Project Collection_ is. The terminology is wibbly wobbly in my opinion even on Microsoft's docs, so it's unclear.

**Section 4**
Project is what I figured is followed by `https://dev.azure.com/{Project Collection}/{Project}`

**Section 5**
Team was hard to guess because I didn't see any teams I was a member of, so I guessed at a Team I may accidentally have access to.

**Section 6**
Usernames, I just used my login id, which case was `pdudley`. If this needs to be a full email, that might need to be specified in the docs.

**Section 7**
Personal Access Token, or PAT. Pretty self-explanatory. However, the permissions for the PAT may need to be listed so as to allow this plugin to function properly.

**Section 8**
Column Names, I just left this as default.

**Section 9**
I chose a folder I had created to keep my Vault root clear. I accidentally left it in my screen shot.

**Section 10**
This is the template for the cards and work items when looking at them. I also left this as default until I get a feel for what I need.

## What's the actual issue here?

![image](https://github.com/BoxThatBeat/obsidian-agile-task-notes/assets/130775229/a4eefb24-8b8a-432b-ab53-cd564eb9aa15)

When I fill in the sections and click refresh, it says that it `Updated Successfully!`, then pops up with an error above and no tasks are presented. How should this actually be configured?

**Comment by BoxThatBeat** [comment, marked as answer]
@symplrdudley You shouldn't have the https:// part in the instance name. Just `dev.azure.com` . In hindsight I should have just had the URL be the only input required. Let me know if this was this issue

**Reply by symplrdudley** [reply]
I'm at least getting a 404 now! So, I need to fix my other details as well.

**Reply by symplrdudley** [reply]
I guess the last question would be what Scopes does my PAT need to have in order to grab what it needs as I'm now getting a 401

**Reply by BoxThatBeat** [reply]
I have not yet tested anything other than full access PATs so I can't answer this question yet. Are you able to use one of those?

**Reply by symplrdudley** [reply]
Well, the 401 was a single anomaly, and have been getting 404's since. Is there any more logging that can be shown in the Console?

**Reply by symplrdudley** [reply]
'twould seem I need to get my permissions straightened out for the DevOps Project I'm attempting to use. Thank you for your help @BoxThatBeat
